common/changes/@itwin/core-backend/common-decoupling_2022-11-03-12-35.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-backend",
+      "comment": "Removing node (runtime) dependencies from core-common",
```

Comment:
can you update the comment here to reflect the changes you made in core-backend?